pom.xml

```diff
@@ -335,12 +335,6 @@
         <artifactId>grpc-stub</artifactId>
         <version>${grpc.version}</version>
       </dependency>
-      <!-- Netty 4.0.50+ required for shading in client runtime jar to work -->
-      <dependency>
-        <groupId>io.netty</groupId>
-        <artifactId>netty-all</artifactId>
-        <version>4.1.30.Final</version>
```

Comment:
Should we remove this since our dependencies pull netty and we may want to fix the version?

Reply:
Why do we need this for shading to work? @apc999 ?

Reply:
- first, this is in `<dependencyManagement>` and I don't think version set here affect transitive dependencies
- regarding why we need this for shading to work, this is actually a fix by calvin https://github.com/Alluxio/alluxio/pull/6497 . w.r.t. that PR, should we also remove this line https://github.com/Alluxio/alluxio/blob/master/core/client/runtime/pom.xml#L90 , @calvinjia

Reply:
Some dependencies might be bringing these as well. I am for not touching the shading directives.

Reply:
@calvinjia  Ping

Reply:
@apc999 Waiting on response for question above:
Isn't dependency management also used to manage the version brought in by the transitive dependencies? For example if A and B depend on Netty 1.0 and Netty 1.4, we can use dependency management to enforce the version used instead of implicitly (based on order of declaration/dependency depth).

Reply:
@calvinjia Right. Maven will enforce transitive dependency version.
At current state, without this direction we're pulling 2 versions: 4.1.30 for grpc, and 4.1.29 for zookeeper
With this directive we can unify the versions.
I'm reverting this directive.